internal/v1/handler.go

```diff
@@ -438,12 +452,18 @@ func (h *Handlers) GetComposes(ctx echo.Context, params GetComposesParams) error
 	if params.Offset != nil {
 		offset = *params.Offset
 	}
+	ignoreImageTypeStrings := ConvertIgnoreImageTypeToSlice(params.IgnoreImageTypes)
 
 	// composes in the last 14 days
-	composes, count, err := h.server.db.GetComposes(idHeader.Identity.OrgID, (time.Hour * 24 * 14), limit, offset)
+	composes, count, err := h.server.db.GetComposes(idHeader.Identity.OrgID, (time.Hour * 24 * 14), limit, offset, ignoreImageTypeStrings)
 	if err != nil {
 		return err
 	}
+	// Convert ignoreImageTypeStrings to []ImageTypes
+	ignoreImageTypes := make([]ImageTypes, len(ignoreImageTypeStrings))
```

Comment:
do we need this variable ?

Reply:
no, I put it here, becuase I thought its good to add it to the composeResponse